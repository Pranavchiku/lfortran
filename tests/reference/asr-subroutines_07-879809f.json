{
    "basename": "asr-subroutines_07-879809f",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/subroutines_07.f90",
    "infile_hash": "8467217fadf16e3376047c00527af5bf50d1bf7077f84d1e380c393c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-subroutines_07-879809f.stdout",
    "stdout_hash": "a2e5b86dcb9b1d159270757885010d7a48cc16846331a067dfd5a7db",
    "stderr": "asr-subroutines_07-879809f.stderr",
    "stderr_hash": "b33a2a2336733ec50d7132471ce66628cc7bd7f8eb19bf93034cef37",
    "returncode": 0
}