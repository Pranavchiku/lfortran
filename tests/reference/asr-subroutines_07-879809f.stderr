warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:10:9
   |
10 |         integer, intent(in) :: a, d, c
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:10:9
   |
10 |         integer, intent(in) :: a, d, c
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:10:9
   |
10 |         integer, intent(in) :: a, d, c
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:11:9
   |
11 |         integer, intent(in), optional :: kind
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:12:9
   |
12 |         integer, intent(out) :: b
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:21:5
   |
21 |     integer :: i, j, k, l
   |     ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:21:5
   |
21 |     integer :: i, j, k, l
   |     ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:21:5
   |
21 |     integer :: i, j, k, l
   |     ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/subroutines_07.f90:21:5
   |
21 |     integer :: i, j, k, l
   |     ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
